todo/src/Todo.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+import './TodoList.css'
+
+import { connect } from 'react-redux';
+import { toggle } from './actions';
+import { removeTodo } from './actions';
+
+const Todo = (props) => {
+
+  return(
+    <div>
+      <li style={props.completed ? {textDecoration: 'line-through'} : null} onClick={() => {props.toggle(props.id)}}>{props.todo}</li>
+      <button onClick={() => {props.removeTodo(props.id)}}>x</button>
+    </div>
+  );
+}
+
+const mapStateToProps = state =>{
+  return{
+    todos: state.todos
+  }
+}
+
+export default connect(mapStateToProps, { toggle: toggle, removeTodo: removeTodo })(Todo);
```

Comment:
How could you handle this component without using redux? Could you pass the action creators down from the parent component as props? What happens if you do that?